homeassistant/components/notify/file.py

```diff
@@ -48,7 +52,7 @@ def send_message(self, message="", **kwargs):
                     '-' * 80)
                 file.write(title)
 
-            if self.add_timestamp == 1:
+            if self.add_timestamp:
                 text = '{} {}\n'.format(dt_util.utcnow().isoformat(), message)
                 file.write(text)
```

Comment:
It's not your code, but a suggestion is to move `file.write(text)` to below the `if: ... else: ...` block and delete the other one.

It looks like it can take coverage up to 96% [see here](https://coveralls.io/builds/7677838/source?filename=homeassistant%2Fcomponents%2Fnotify%2Ffile.py#L57)